src/player.cpp

```diff
@@ -10102,6 +10102,20 @@ int player::get_wind_resistance(body_part bp) const
     return totalCoverage;
 }
 
+int player::get_weight() const
```

Comment:
This is a good point, I was originally planning on using player::inv_dump() to get a list of all items to weigh, but ended up basically reimplimenting it const style. It really belongs in `character` though, it's true -- it could later be used to get monster weight that way.